apps/fabric-website-resources/package.json

```diff
@@ -53,7 +53,8 @@
     "react-highlight": "0.10.0",
     "react-test-renderer": "^16.3.0",
     "resemblejs": "~2.2.3",
-    "sinon": "^4.1.3"
+    "sinon": "^4.1.3",
+    "@uifabric/build": "*"
```

Comment:
`@uifabric/build` is unpublished. Will referencing it in devDependencies of a published project cause errors?

Reply:
No, you won't install devDeps when you take fabric-website-resources as a dep. This should be fine.